released/packages/coq-iris-string-ident/coq-iris-string-ident.0.1.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "coq-iris-string-ident"
+maintainer: "Ralf Jung <jung@mpi-sws.org>"
+authors: "Tej Chajed <tchajed@mit.edu>"
+license: "BSD"
```

Comment:
```suggestion
license: "BSD-3-Clause"
```
We want SPDX identifiers for licenses. Can you confirm that this is what is referred to: https://spdx.org/licenses/BSD-3-Clause.html

Reply:
Specifically we used https://opensource.org/licenses/BSD-3-Clause but that seems to be the same.